developerTools/scriptReferences/entityClientScript/entityClientScript.js

```diff
@@ -0,0 +1,116 @@
+//
+// entityClientScript.js
+//
+// Adapted from script https://github.com/robin-k-wilson/hifi-dev-utils/blob/master/client_entity_all_events.js
+// Updated by Robin Wilson 4/15/2019
+//
+// Generic client entity methods and signals that are triggered by mouse or controller interactions
```

Comment:
please move these comments into a README. the README should also have the normal release notes header.

Reply:
Got it!